jupyter_foundation.md

```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6) whose purpose is to raise, budget and spend funds in support of Project Jupyter and its mission. The official Jupyter Foundation charter is here (TODO: link). A brief summary of relevant points from the charter is provided below.
+
+
+## Jupyter Foundation Governing Board
+
+The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).
+
+## Responsibilities of the Jupyter Foundation Governing Board
+
+- Serve Project Jupyter and its mission.
```

Comment:
To copy language from the draft Jupyter Foundation charter, let's change "Serve" to "Support"
```suggestion
- Support Project Jupyter and its mission.
```

Reply:
:+1: